backend/middlewares/validators/lessonValidators.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response, NextFunction } from "express";
+import {
+  getApiValidationError,
+  validateArray,
+  validatePrimitive,
+} from "./util";
+import { getErrorMessage } from "../../utilities/errorUtils";
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable-next-line import/prefer-default-export */
+export const courseRequestDtoValidator = async (
```

Comment:
`lessonRequestDtoValidator`? Also I'm not sure you have a `LessonRequestDTO` right now, only `LessonDTO`